### PR TITLE
Javadoc generation

### PR DESCRIPTION
**Pull request by alexander-aleksandrov**
Fix all Javadoc errors and ensure that we have generated Javadocs.

### CODE REVIEW

**Comment by alexander-aleksandrov** [comment]
@dmdashenkov PTAL

File: server/src/main/java/org/spine3/server/entity/storage/Column.java [hunk 1]
```diff
@@ -42,8 +42,8 @@
  *
  *
  * <p>These methods represent the Columns:
- * <code>
- *      <pre>
+ * <pre>
+ *      <code>
```
Comment (dmdashenkov) [inline server/src/main/java/org/spine3/server/entity/storage/Column.java]: Please prefer `{@code ... }` to `<code> ... </code>` here and in other similar places.

**Comment by alexander-aleksandrov** [comment]
@dmdashenkov PTAL

File: testutil/src/main/java/org/spine3/test/Verify.java [hunk 2]
```diff
@@ -1508,7 +1508,9 @@ public static void assertThrows(
      * <p>
      * e.g.
      * <pre>
-     * Verify.assertThrowsWithCause(RuntimeException.class, IOException.class, new Callable<Void>()
+     * Verify.assertThrowsWithCause(RuntimeException.class,
+     *                              IOException.class,
+     *                              new Callable&lt;Void&gt;()
```
Comment (dmdashenkov) [inline testutil/src/main/java/org/spine3/test/Verify.java]: Please fix the code formatting in this snippet regarding the Spine code style conventions.

**Pull request merged**